build/buildall

```diff
@@ -47,8 +47,12 @@ function print_usage() {
   echo "        Build in parallel, N (4 by default) is passed via -P to xargs"
   echo "   --install"
   echo "        Intall the resulting jar instead of just building it"
+  echo "   -o=MVN_OPT, --option=MVN_OPT"
+  echo "        use this option to build project with maven. E.g., --option='-Dcudf.version=cuda11'"
```

Comment:
Rather than needing to specify `-o` for every Maven option, why not simply assume that any argument encountered that isn't recognized as an option is the start of the arguments to pass through to Maven?

Reply:
Do you mean pass through args like `mvn ... $*`?  We have below block to mark `Unknow arg`,  
```
*)
  echo >&2 "Unknown arg: $1"
  print_usage
  exit 1
  ;;
```

`-o` seems more friendly than gathering all the unknow options as mvn opts.

and it can pass all options by `-o="-Dabc=1 -Pdef=2 ... `

Reply:
Requiring the user to prefix every mvn option with `-o=` is annoying.  And as far as unknown arg, Maven will either process it as a useful argument or complain itself.  Given the whole point of this buildall script is to wrap invocations of Maven, what other use would there be for extra command-line arguments than pass them to Maven?

I'm not going to block the PR over this, but I find it odd saying it's "more friendly" for the user to jump through extra hoops here.